src/modules/keyboardmanager/ui/EditKeyboardWindow.cpp

```diff
@@ -175,7 +175,7 @@ void createEditKeyboardWindow(HINSTANCE hInst, KeyboardManagerState& keyboardMan
     HWND _hWndEditKeyboardWindow = CreateWindow(
         szWindowClass,
         L"Remap Keyboard",
-        WS_OVERLAPPEDWINDOW | WS_VISIBLE,
+        WS_OVERLAPPED | WS_CAPTION | WS_SYSMENU | WS_THICKFRAME | WS_MAXIMIZEBOX,
```

Comment:
I don't think that removing the minimize button is that useful, since you still can interact with the main Settings window and it just makes it a little bit harder for the user to navigate to some other window.

Reply:
That's true, but we do want it to feel like the user should close it before doing anything else like an actual dialog box. Since that doesnt seem to be possible to actually enforce it since they are two separate processes, this was the minimum step we can do. Like you said users will feel like closing it if they cant minimize it which is exactly what we want the users to do if they want to switch to something else.